src/custom/state/orders/updater.ts

```diff
@@ -187,7 +189,11 @@ export function EventUpdater(): null {
           // Get order from our store
           // and from the backened API
           const orderFromStore = findOrderById(id)
-          const orderFromApi = await getOrder(chainId, id)
+          // We've found the orderId in the Trade event
+          // But the backend may not be completely updated yet, e.g
+          // the frontend is ahead of the backend in regards to data freshness
+          // TODO: temporary! change to a better solution
```

Comment:
Add the link to the issue, I think we have one already. Anyways, unrelated these changes to this PR

Reply:
ok will add